### PR TITLE
Initialize scale_ in TensorizedCovarianceModel constructor

### PR DESCRIPTION
**Pull request by dbarbier**
Fix trac#870.

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -38,6 +38,7 @@
 === Miscellaneous ===
 
 === Bug fixes ===
+ * #870 (Problem with TensorizedCovarianceModel with spatial dim > 1)
```
Comment (jschueller) [inline ChangeLog]: could you move this change to the 1.10 section ? I'll likely backport that fix

Comment (dbarbier) [reply ChangeLog]: IMO it is too late, we do not know whether kriging works in this case, this can wait for 1.11.

Comment (jschueller) [reply ChangeLog]: ok

File: lib/src/Base/Stat/TensorizedCovarianceModel.cxx [hunk 2]
```diff
@@ -46,6 +46,7 @@ TensorizedCovarianceModel::TensorizedCovarianceModel(const CovarianceModelCollec
   : CovarianceModelImplementation()
 {
   setCollection(collection);
+  scale_ = Point(inputDimension_, 1.0);
   activeParameter_ = Indices(getScale().getSize() + getAmplitude().getSize());
```
Comment (jschueller) [inline lib/src/Base/Stat/TensorizedCovarianceModel.cxx]: pyinstallcheck_CovarianceModel_std's output must be updated

**Comment by regislebrun** [comment]
+1

**Pull request merged**